pkg/vm/emit/emit.go

```diff
@@ -156,13 +157,19 @@ func AppCall(w *io.BinWriter, scriptHash util.Uint160) {
 	Syscall(w, interopnames.SystemContractCall)
 }
 
-// AppCallWithOperationAndArgs emits an APPCALL with the given operation and arguments.
-func AppCallWithOperationAndArgs(w *io.BinWriter, scriptHash util.Uint160, operation string, args ...interface{}) {
+// AppCallWithFlag emits an APPCALL with the full set of parameters.
+func AppCallWithFlag(w *io.BinWriter, scriptHash util.Uint160, operation string, f flags.CallFlag, args ...interface{}) {
```

Comment:
Do we really need `emit.AppCall`? I think this can be a substitute for it.

Reply:
Fixed.